packages/core/upload/admin/src/utils/getFolderURL.js

```diff
@@ -1,6 +1,11 @@
 import { stringify } from 'qs';
 
 const getFolderURL = (pathname, query, folder) => {
+  // Content manager do not use pathname
+  if (!pathname) {
+    return folder ? folder.id : null;
+  }
+
```

Comment:
I'd prefer if we not do that: the function is intended to return a folder url (string) and not an id or `null`. In case this does not work in the content-manager the function shouldn't be called there at all (if possible).

What do you think?

Reply:
I agree I'm not comfortable with that either - I did it because `getBreadcrumbData` is also needed in the content-manager where we don't want a proper URL but only the id or null (in case it's to navigate to the ML root).
I started by integrating this tricky logic to `getBreadcrumbData` directly but found it a bit messy - but maybe it is still better than that, wdyt?

Reply:
Mh, what I'd propse instead then: you either 

- pass an empty string as `pathname`, so that you don't have to change the signature of the function and in your component you just do not use the URL if you render a button
- you pass a param to `getBreadcrumbData` indicating that it does not need to return the URL
- you implement another version of `getBreadcrumbData` for the CM which only returns what you need

Wdyt?

Reply:
my first thought was to have separated `getBreadcrumbDataML` and `getBreadcrumbDataCM` functions, I think it would be more readable for the future developers - I first thought it would be a shame to create a second one but if you think it's okay I would go for it, wdyt?

Reply:
I agree with your doubts, but I think the issue stems from the fact that both media libraries (the main one and the one in the modal) work so different. I'd prefer to have two utilities instead of having one with a shortcut :)